htdocs/core/triggers/interface_50_modBlockedlog_ActionsBlockedLog.class.php

```diff
@@ -119,7 +119,7 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			$amounts = 0;
 			if (!empty($object->amounts)) {
 				foreach ($object->amounts as $amount) {
```

Comment:
$amount here is a variable in memory. So its content should already be a valid amount.
So I think we should not have price2num() at all here.
We should just have 
$amounts += (double) amount;
like we have into other trigger codes of the same trigger.
What did you get into the $object->amount array ?